google/cloud/aiplatform/datasets/datasources.py

```diff
@@ -0,0 +1,141 @@
+import abc
```

Comment:
This module should be prefixed with underscore,  ie: `_datasources` to indicate that we don't intend for it to be used on the API surface.